internal/observability/event/cloudevents_formatter_node.go

```diff
@@ -69,12 +69,13 @@ func newCloudEventsFormatterFilter(source *url.URL, format cloudevents.Format, o
 			}
 			n.deny = append(n.deny, f)
 		}
+	} else {
```

Comment:
@jimlambrt This is the place where I am checking if there are no user-specified `deny` filters, we add the default `deny` filter. since it is in `else` for the case that the user specifies the `deny` filter. We no longer have this default filter. I wanted to double-check the logic with you.
Another scenario could be the user specifying the `deny` filter when the observation event is enabled which with this logic, we will override the default again and we emit the `ServerCoordinationService/Status`

Reply:
I wasn't aware that we were already filtering ServerCooridnationService msgs out
of audit events.  

Given that, I'm not sure our change is quite right.  Perhaps
we should add `"/type" contains "observation"` to the original default filter,
so it only applies to audit events and we should always add that filter because
that's the current behavior.  

Then we can add the conditional addition of the new observation filter.  Make sense?

Reply:
Would that be `"/type" contains "audit"` on the first one?

Reply:
So, in short... The existing default filter path was incorrectly defined, and didn't work? Which would mean we haven't been auto-filtering previously... 

Technically then, default adding the observation filter if no other deny filters exist would be correct I think (deferring to @jimlambrt here), but also raises the question of whether or not a ticket needs to be raised within backend to fix the behaviour for audit events.

Reply:
If the exiting path was broken, then you can just ignore my comment/concern.  Your changes are perfectly fine.

Reply:
I just confirmed against `main` that this is indeed the case. Running `boundary dev -audit-events true` and seeing the  status events coming through.